packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -43,6 +43,9 @@ export class DeltaScheduler {
     // to process the ops.
     private schedulingCount: number = 0;
 
+    // Keeps track of the Queue length before processing a batch.
+    private queueLengthBeforeBatchBegin: number = 0;
```

Comment:
I think this should be called `numberOfOpsToProcess` or something similar since we are interested in ops.

Reply:
removed it.